lesion_analysis/lesion_seg_analysis.py

```diff
@@ -0,0 +1,150 @@
+"""
```

Comment:
apparently this file is not used anymore-- so, consider removing. Also, consider discussing why not using `sct_analyse_lesion`. And if the SCT function has issues (eg: slow, etc.), consider modifying `sct_analyse_lesion` instead.

Reply:
Also see https://github.com/ivadomed/canproco/issues/38#issuecomment-1650648905